app/views/map/_mapDependencies.html.erb

```diff
@@ -18,4 +18,4 @@
 <%= javascript_include_tag('https://cdnjs.cloudflare.com/ajax/libs/OverlappingMarkerSpiderfier-Leaflet/0.2.6/oms.min.js') %>
 
 <%= javascript_include_tag('/lib/leaflet-environmental-layers/example/lib/glify.js') %>
-
+<%= javascript_include_tag('/lib/leaflet-environmental-layers/example/lib/leaflet-fullUrlHash.js') %>
```

Comment:
Same here but change to `<%= javascript_include_tag('/lib/leaflet-environmental-layers/lib/leaflet-fullUrlHash.js') %>`

Reply:
Hi @pythongiant. in #7029, I did this and received the same error message. I updated to use 'example' in the link, as seen in the original line #20, to try overcome the error message. Whilst this left the same error message,  I've just updated the code in line with the task sample.